schemas/2018-01-22/Microsoft.Devices.json

```diff
@@ -71,6 +71,32 @@
         "name"
       ],
       "description": "Microsoft.Devices/IotHubs"
+    },
+    "IotHubs_eventHubEndpoints_ConsumerGroups": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Devices/IotHubs/eventHubEndpoints/ConsumerGroups"
+          ]
+        },
+        "apiVersion": {
```

Comment:
>apiVersion [](start = 9, length = 10)

you don't have a "properties" bag?

Reply:
Swagger has this as Consumer group info which contains the property bags.
"EventHubConsumerGroupInfo": {
      "description": "The properties of the EventHubConsumerGroupInfo object.",
      "x-ms-azure-resource": true,
      "type": "object",
      "properties": {
        "properties": {
          "description": "The tags.",
          "type": "object",
          "additionalProperties": {
            "type": "string"
          }
        },
        "id": {
          "description": "The Event Hub-compatible consumer group identifier.",
          "type": "string",
          "readOnly": true
        },
        "name": {
          "description": "The Event Hub-compatible consumer group name.",
          "type": "string",
          "readOnly": true
        },
        "type": {
          "description": "the resource type.",
          "type": "string",
          "readOnly": true
        },
        "etag": {
          "description": "The etag.",
          "type": "string",
          "readOnly": true
        }
      }
    }
Does this object have some issue as well?